cip-00XX/cip-00XX.md

```diff
@@ -0,0 +1,146 @@
+- A third trigger, `UnclaimedActivityRecordExpiryTrigger`, expires unused `UnclaimedActivityRecord` contracts.
+
+### Minting Mechanism
+
+- `UnclaimedActivityRecord` is introduced as a new transferable on-ledger asset.
+- It is added as a new variant (`InputUnclaimedActivityRecord`) within the existing `TransferInput` type.
+- This input is consumed by `AmuletRules_Transfer`, which mints the corresponding Amulet during a self-transfer executed by automation in the Wallet App (`CollectRewardsAndMergeAmuletsTrigger`).
+
+### Off-ledger Implementation Requirements
+
+- **SV App**
+  - Implement and deploy the three new triggers listed above.
+  - Extend the Treasury Service to support the new activity record lifecycle.
+
+- **Wallet App**
+  - Extend the `CollectRewardsAndMergeAmuletsTrigger` to auto-collect `UnclaimedActivityRecord` using the new transfer input.
+
+- **CLI App**
+  - Introduce a Python CLI utility to calculate minted and unclaimed rewards by party ID and time range, reusing logic from `scan_txlog.py`.
+
+- **SV App Voting UI**
+  - Update the SV App voting UI to support proposals using `DsoRules_CreateUnallocatedUnclaimedActivityRecord`.
+
+- **SV App Governance UI**
+  - Add support in the SV App for configuring `unallocatedUnclaimedActivityRecordTimeout`.
+
+## Motivation
+
+This change is required to implement the reward distribution mechanism described in CIP-0058. 
+The CIP defines a process whereby, upon successful milestone verification, the Tokenomics Working Group approves a 
+reward amount and the Super Validators assign a portion of unclaimed rewards to be minted. 
+This functionality leverages the standard voting process and introduces new ledger, off-ledger, and UI components to 
+support reward calculation, archival of unclaimed rewards, and minting of the corresponding Amulet.
+
+This aligns with the first deliverable of [CIP-0058](https://github.com/global-synchronizer-foundation/cips/blob/main/cip-0058/cip-0058.md): _Enable escrow mechanisms to ensure Super Validators deliver value in return for rewards_.
+
+
+## Rationale
+
+This proposal introduces a governance-backed, automated reward allocation mechanism that ensures Super Validators (SVs) never act as direct issuers of Amulets. Instead, it relies on existing voting infrastructure, explicit reward tracking, and on-ledger automation. The design satisfies the CIP-0058 objective of enabling escrowed value transfer with network approval, while preserving clear separation of duties between reward authorization and execution.
+
+Super Validators need a way to reward parties for milestone-based work without directly issuing Amulets. To meet this goal, the system must:
+
+- Let SVs propose reward allocations through a formal vote.
+- Track and escrow those rewards in a verifiable, on-ledger structure.
+- Allow beneficiaries to receive their rewards without needing manual approval or signature from SVs.
```

Comment:
not quite clear to me what you mean here. In the end the SVs do explicitly approve by creating the vote.